src/runtime/serialization/SerializedValue.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2021-present Samsung Electronics Co., Ltd
+ *
+ *  This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Lesser General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 2.1 of the License, or (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public
+ *  License along with this library; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301
+ *  USA
+ */
+
+#ifndef __EscargotSerializedValue__
+#define __EscargotSerializedValue__
+
+#include "runtime/Value.h"
+
+namespace Escargot {
+
+class SerializedValue {
+    friend class Serializer;
+
+public:
+    enum Type {
+        Undefined,
+        Null,
+        Boolean,
+        Number,
+        String,
+        BigInt
```

Comment:
Is this not used yet?

Reply:
Yes not used yet. I have new commit for it.
but I can remove it now